### PR TITLE
feat(language): update czech language

### PR DESCRIPTION
**Pull request by Smexhy**
#### Description

updates the czech language from overseer weblate and add jellyseer specifics

### CODE REVIEW

File: src/i18n/locale/cs.json [hunk 1]
```diff
@@ -410,7 +410,7 @@
   "components.Settings.SettingsJobsCache.cachemisses": "Neúspěchy",
   "components.NotificationTypeSelector.usermediadeclinedDescription": "Dostat oznámení o odmítnutí vašich požadavků na média.",
   "components.NotificationTypeSelector.usermediaavailableDescription": "Dostat oznámení, jakmile budou k dispozici žádosti o média.",
-  "components.Layout.VersionStatus.commitsbehind": "{commitsBehind} {commitsBehind, plural, one {commit} other {commits}} za",
+  "components.Layout.VersionStatus.commitsbehind": "{commitsBehind} {commitsBehind, plural, one {commit} other {commitů}} za",
```
Comment (fallenbagel) [inline src/i18n/locale/cs.json:413]: Why is the {{commits}} changed?

File: src/i18n/locale/cs.json [hunk 2]
```diff
@@ -539,5 +539,508 @@
...
+  "components.UserProfile.UserSettings.UserNotificationSettings.pushoverApplicationTokenTip": "<ApplicationRegistrationLink>Registrace aplikace</ApplicationRegistrationLink> pro použití s {applicationTitle}",
+  "components.UserProfile.UserSettings.UserNotificationSettings.telegramsettingssaved": "Nastavení oznámení Telegramu úspěšně uloženo!",
+  "components.UserProfile.UserSettings.UserNotificationSettings.webpushsettingsfailed": "Nastavení webových push oznámení se nepodařilo uložit.",
+  "components.UserProfile.UserSettings.UserPasswordChange.noPasswordSet": "Tento uživatelský účet v současné době nemá nastavené heslo. Níže nastavte heslo, aby se tento účet mohl přihlašovat jako \"místní uživatel.\"",
+  "components.UserProfile.UserSettings.UserPasswordChange.noPasswordSetOwnAccount": "Váš účet v současné době nemá nastavené heslo. Níže nastavte heslo, abyste se mohli přihlásit jako \"místní uživatel\" pomocí své e-mailové adresy.",
+  "i18n.importing": "Importování…",
+  "components.IssueList.IssueItem.episodes": "{episodeCount, plural, zero {Epizod} one {Epizoda} other {Epizody}}",
```
Comment (fallenbagel) [inline src/i18n/locale/cs.json:1013]: Here too {} ones should remain as they were

File: src/i18n/locale/cs.json [hunk 3]
```diff
@@ -539,5 +539,508 @@
...
+  "components.IssueList.IssueItem.seasons": "{seasonCount, plural, zero {Sezón} one {Sezóna} other {Sezóny}}",
```
Comment (fallenbagel) [inline src/i18n/locale/cs.json:1014]: this one too

**Review by Fallenbagel — CHANGES_REQUESTED** [review]
~From 1014 - 1045 you (i did not add comment) but all the {} ones should not be changed~
Ignore this.

File: src/i18n/locale/cs.json (hunk 3 above)
Comment (Smexhy) [reply src/i18n/locale/cs.json:1014]: I have used the following syntax https://formatjs.io/docs/core-concepts/icu-syntax#plural-format translating multiple forms of plurals. The strings inside mentioned brackets need to be translated as well. For example it was translated that way in german localisation file as well: https://github.com/Fallenbagel/jellyseerr/blob/883b9377be209de534db0a7bc9e56a125e3df39a/src/i18n/locale/de.json#L997

Comment (fallenbagel) [reply src/i18n/locale/cs.json:1014]: Ah sorry. I am not familiar with that. Sorry for taking your time

**Review by Fallenbagel — APPROVED** [review]

**Comment by Fallenbagel** [comment]
@all-contributors please add @Smexhy for translation

**Pull request merged**